packages/core/src/event_delegation_utils.ts

```diff
@@ -45,9 +47,9 @@ export function setJSActionAttributes(nativeElement: Element, eventTypes: string
 }
 
 export function setJSActionAttribute(nativeElement: Element, eventType: string) {
-  const existingAttr = nativeElement.getAttribute(Attributes.JSACTION);
-  //  This is required to be a module accessor to appease security tests on setAttribute.
-  nativeElement.setAttribute(Attributes.JSACTION, `${existingAttr ?? ''}${eventType}:;`);
+  const cache = getCache(nativeElement) || [];
+  cache[eventType] = '';
+  setCache(nativeElement, cache);
```

Comment:
I'm wondering if we can expose a different API from JsAction to add and remove event types (vs pulling a cache and making changes to it), something like this:
```suggestion
  registerEventType(nativeElement, eventType);
  // ... later on ...
  unregisterEventType(nativeElement, eventType);
```
This would make cache structure an implementation detail (so it'd be easier to update).

Reply:
done